src/Illuminate/Contracts/Routing/ResponseFactory.php

```diff
@@ -14,6 +14,15 @@
      */
     public function make($content = '', $status = 200, array $headers = []);
 
+    /**
+     * Return an empty response.
+     *
+     * @param  int  $status
+     * @param  array  $headers
+     * @return \Illuminate\Http\Response
+     */
+    public function noContent($status = 204, array $headers = []);
```

Comment:
I think it will be better to put it into 5.7.
Because we added one more method to the contract.

---

however without contract changing it can be target to the 5.6